thispastwinter/To-Do
# JKA-20: Initial App Styling

## Description

Initial Styling based on Sean's InVision design.

### 📋 Acceptance Criteria

> Checked off by the PR **Assignee(s)**

- [ ] CRITERION_HERE


### 👩‍🔬 Test Instructions

1. Run 'npm start'.
2. Check to see that the design meets the standards of Sean's Prototype.

## 🔎 Reviewer Checklist

> Checked off by the PR **Reviewers**

### Required

> These always need to be checked


### Additional

> Delete if unneeded

- [ ] Code has been tested and confirmed locally

---

>### Roles & Responsibilities
>
>#### 👨‍💻 Justin Klaas
>
>- Initiator of this PR (be sure to set in GitHub UI)
>- Addresses feedback and change requests
>- Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
>#### 👩‍💻 Blake Tarter
>
>- Invited to review PR by **Assignee** (via GitHub UI)
>- Is expected to complete a review and address followup
